anomaly/config.cpp

```diff
@@ -31,6 +31,11 @@ unsigned int Config::get_delete_cost()
     return _delete_cost;
 }
 
+unsigned int Config::get_relabel_cost()
+{
+    return _relabel_cost;
+}
+
 unsigned int Config::get_relabel_cost(const std::string& src, const std::string& dst)
```

Comment:
I'm not sure if this method should be in config class. IMO config should just keep the configuration, and any other calculations should be defined in a place where you actually need it.

Reply:
I changed it like this because of this comment: https://github.com/amzn/hawktracer/pull/32/files/ababef96faa1fe90af29db6748d0e912561209fb#r196979047
Should I move the method in the OrderedTreeEditingDistance class then?

Reply:
Yes, I think it's better if config doesn't have any logic related to editing distance algorithm